open_ce/build_env.py

```diff
@@ -97,7 +97,7 @@ def build_env(args):
 
     # If repository_folder doesn't exist, create it
     if args.repository_folder and not os.path.exists(args.repository_folder):
-        os.mkdir(args.repository_folder)
+        os.makedirs(args.repository_folder, exist_ok=True)
```

Comment:
Can remove the exists check in the if statement now.

Reply:
Ah, yeah. I made this change because it would fail if the parent folder didn't exist.

Reply:
I went ahead and updated the other locations too.